cristin-import/src/test/java/no/unit/nva/cristin/lambda/FakeEventBridgeClient.java

```diff
@@ -30,8 +35,19 @@ public void close() {
 
     }
 
-    public List<PutEventsRequest> getEventRequests() {
-        return evenRequests;
+    @Override
+    public ListEventBusesResponse listEventBuses(ListEventBusesRequest listEventBusesRequest)
```

Comment:
In line 76 in EventEmitter.java, we make an explicit search for the bus that we are about to write into. This creates the need to mock the respective functionality in our FakeEventBridgeClient. The mocking is returning a (singleton) list of the supplied eventbus name